devtools-ci-index.yaml

```diff
@@ -657,6 +657,78 @@
             exit $rtn_code
 
 - job:
+    name: 'devtools-che-test-end-to-end'
+    defaults: global
+    node: devtools
+    properties:
+        - github:
+            url: https://github.com/redhat-developer/che-e2e-tests
+    scm:
+        - git:
+            url: https://github.com/redhat-developer/che-e2e-tests.git
+
+            shallow_clone: true
+            branches:
+                - master
+    triggers:
+        - github
+        - github-pull-request
+            white-list:
+                - mlabuda
+            trigger-phase: '[testPR]'
+            white-list-target-branches:
+                - master
+    wrappers:
+        - credentials-binding:
+            - text:
+                credential-id: de793dce-5821-4182-af4f-6c7b42b5c21d
+                variable: KEYCLOAK_TOKEN
+    builders:
+        - shell: |
+            set +e
+            set +x
+            env > jenkins-env
+            cat $creds_config_file >> jenkins-env
+            export CICO_API_KEY=$(cat ~/duffy.key )
+            # get node
+            n=1
+            while true
+            do
+                cico_output=$(cico node get -f value -c ip_address -c comment)
+                if [ $? -eq 0 ]; then
+                    read CICO_hostname CICO_ssid <<< $cico_output
+                    if  [ ! -z "$CICO_hostname" ]; then
+                        # we got hostname from cico
+                        break
+                    fi
+                    echo "'cico node get' succeed, but can't get hostname from output"
+                fi
+                if [ $n -gt 5 ]; then
+                    # give up after 5 tries
+                    echo "giving up on 'cico node get'"
+                    exit 1
+                fi
+                echo "'cico node get' failed, trying again in 60s ($n/5)"
+                n=$[$n+1]
+                sleep 60
+            done
+            echo 'Using Host' $CICO_hostname
+            set -x
+            sshopts="-t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root"
+            ssh_cmd="ssh $sshopts $CICO_hostname"
+            $ssh_cmd yum -y install rsync
+            rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
+            $ssh_cmd -t "cd payload && /bin/bash cico_run_EE_tests.sh"
+            rtn_code=$?
+            cico node done $CICO_ssid
+            exit $rtn_code
+    publishers:
+        - archive:
```

Comment:
yeah, I could remove it, as well as storing target/*. For successful execution of tests it is not required. 

Does publishers do not work at all or only this particular one?